app/views/feedback/steps/_satisfied.html.erb

```diff
@@ -0,0 +1,2 @@
+
+<%= f.govuk_select :state, f.object.class::OPTIONS, options: { prompt: "Please select" }, label: { text: "Overall, how satisfied were you with your experience on our website?" } %>
```

Comment:
Can this question (and did you achieve what you wanted from your visit) be on the same page as the other questions, or is this too difficult given these 2 are asked in all circumstances?

Reply:
Can this question be a radio component also?